queue/abstract.lua

```diff
@@ -778,16 +830,28 @@ local function cfg(self, opts)
     opts = opts or {}
     local session_opts = {}
 
+    -- Set default replicaset_mode value
+    if opts['replicaset_mode'] == nil then
+        opts['replicaset_mode'] = false
+    end
+
     -- Check all options before configuring so that
     -- the configuration is done transactionally.
     for key, val in pairs(opts) do
         if key == 'ttr' then
             session_opts[key] = val
+        elseif key == 'replicaset_mode' then
+            if type(val) ~= 'boolean' then
+                error('Invalid value of replicaset_mode: ' .. tostring(val))
+            end
+            session_opts[key] = val
         else
             error('Unknown option ' .. tostring(key))
         end
     end
 
+    migrate_queue_taken_space(opts['replicaset_mode'])
```

Comment:
This is is done so that the call to queue.cfg displays the state of the parameter if it is not set:

```
tarantool> queue.cfg
---
- ttr: 300
  in_replicaset: false
```